cli/cli.ts

```diff
@@ -5434,13 +5518,20 @@ function internalCheckDocsAsync(compileSnippets?: boolean, re?: string, fix?: bo
             }
         });
 
-    function addSnippet(snippet: CodeSnippet, entryPath: string, snipIndex: number) {
+    function addSnippet(snippet: CodeSnippet, entryPath: string, snipIndex: number, src: string) {
+        const key = `${src}${snipIndex}`;
+        if (existingSnippets[key])
```

Comment:
If a file is linked both in the summary and the home page galleries, it was getting checked twice. This cuts out about ~125 snippets from arcade that were getting run twice.